user-interface/src/main/java/life/qbic/datamanager/parser/MetadataConverter.java

```diff
@@ -189,7 +192,7 @@ private List<MeasurementMetadata> convertNGSMeasurement(ParsingResult parsingRes
                   ""))),
           safeListAccess(row.values(), keyIndices.getOrDefault(ORGANISATION_ID.propertyName(), -1),
               ""),
-          safeListAccess(row.values(), keyIndices.getOrDefault(INSTRUMENT.propertyName(), -1), ""),
+          safeListAccess(row.values(), keyIndices.getOrDefault(MS_DEVICE.propertyName(), -1), ""),
```

Comment:
Careful! The NGS Measurement Conversion is using the properties of the ProteomicsMeasurementProperty. This leads to the issue that the instrument column is ignored in the ngs template sheet during registration and editing

Reply:
👀